src/mlang/mpp_ir/mpp_ir_to_bir.ml

```diff
@@ -263,7 +263,7 @@ let generate_verif_call (m_program : Mir_interface.full_program)
       Pos.map_under_mark
         (fun verif_id -> Bir.SRovCall verif_id)
         verif.Mir.cond_number)
-    verifs
+    verifs |> List.rev
```

Comment:
And in case you wonder, this fixes the call order of individual verifications to match the order produced by compir (there is probably a better way to fix this).